scripts/qemu_launcher.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+
+# Copyright (C) 2018 Luxoft
+# All rights reserved
+#
+# This is a handy script used to launch QEMU with different options easily
```

Comment:
What does this sentence mean? I don't understand the "different options easily" part. Does it mean that one can add options on command line which are sent to the command? I think this sentence would benefit from being split up into two or more sentences for readability.

Reply:
I think I understand from reading the code, but I still think that it would be nice from a readability point to rephrase and expand the description. Otherwise the description becomes unnecessary, but I think a description like this is a good introduction, so just expand it slightly.

Reply:
Sorry for the confusion. I add a few more sentences to explain a bit more.